controller/workitem.go

```diff
@@ -256,6 +257,40 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
 	}
+	var space *space.Space
+	err = application.Transactional(c.db, func(appl application.Application) error {
+		// verify spaceID:
+		// To be removed once we have endpoint like - /api/space/{spaceID}/workitems
+		space, err = appl.Spaces().Load(ctx, ctx.SpaceID)
+		if err != nil {
+			log.Error(ctx, map[string]interface{}{
+				"space_id": ctx.SpaceID,
```

Comment:
Oh, I missed the err param.. Will fix that. Regarding the rest.. I just copy-pasted it from the tx below. I just added the log here. Tried to change the code as less as possible. But it makes sense what you are saying.